### DISCUSSION TITLE
misalignment of elements on reload of collab mode excalidraw package

### DISCUSSION THREAD

**Discussion by akashg3627** [excalidraw/excalidraw#3979] [Q&A]
Hello,
I have tried making a collab added version of excalidraw npm package as @akashg3627/excalidraw 
Everything is working smoothly except for one bug coming up. When I reload my Vue app within which the package is integrated as a component, the element gets misaligned to the left top corner. Accurately it positions the center of the element to the left top vertex. Any guidance in this context would be really helpful.

**Comment by dwelle** [comment]
Hey @akashg3627 could you make a reproducible example on https://codesandbox.io?
